ow_plexil/src/plexil-adapter/OwlatAdapter.cpp

```diff
@@ -38,6 +40,335 @@ static void owlat_stow (Command* cmd, AdapterExecInterface* intf)
+                                                CommandId);
+  send_ack_once(*cr);
+}
+
+
+static void armJointAngles (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmJointAngles ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmJointAngles());
+}
+
+static void armJointAccelerations (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmJointAccelerations ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmJointAccelerations());
+}
+
+static void armJointTorques (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmJointTorques ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmJointTorques());
+}
+
+static void armJointVelocities (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmJointVelocities ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmJointVelocities());
+}
+
+static void armFTTorque (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmFTTorque ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmFTTorque());
+}
+
+static void armFTForce (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmFTForce ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmFTForce());
+}
+
+static void armPose (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmPose ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmPose());
+}
+
+static void armTool (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getArmTool ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getArmTool());
+}
+
+static void pspStopReason (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getPSPStopReason ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getPSPStopReason());
+}
+
+static void shearBevameterStopReason (const State& state, StateCacheEntry &entry)
+{
+  debugMsg("getShearBevameterStopReason ", "lookup called for " << state.name()
+           << " with " << state.parameters().size() << " args");
+  entry.update(OwlatInterface::instance()->getShearBevameterStopReason());
+}
+
+static void getDefault (const State& state, StateCacheEntry &entry)
```

Comment:
Could this function have a more descriptive name?  I can't tell what "Default" applies to without more context.

Reply:
I changed it to your suggestion getDefaultLookupHandler. It is in case a lookup is used that does not exist.